sdc/rabbit/publisher.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+from structlog import wrap_logger
+import pika
+
+logger = wrap_logger(logging.getLogger('__name__'))
+
+
+class QueuePublisher(object):
+    """This is a queue publisher that publishes response messages to a
+    RabbitMQ queue.
+
+    """
+    _durable_queue = True
+
+    def __init__(self, urls, queue, **kwargs):
```

Comment:
:param **kwargs: Custom key/value pairs passed to the arguments
            parameter of pika's channel.queue_declare method